zenoh/src/api/key_expr.rs

```diff
@@ -272,6 +272,30 @@ impl<'a> KeyExpr<'a> {
             Ok(r.into())
         }
     }
+
+    pub(crate) fn keyexpr_intersect<'b, L, R>(left: L, right: R) -> bool
+    where
+        L: TryInto<KeyExpr<'a>>,
+        R: TryInto<KeyExpr<'b>>,
+    {
+        if let (Ok(l), Ok(r)) = (left.try_into(), right.try_into()) {
+            l.intersects(&r)
+        } else {
+            false
+        }
+    }
+
+    pub(crate) fn keyexpr_include<'b, L, R>(left: L, right: R) -> bool
```

Comment:
Similar comment as `keyexpr_intersect`.

Reply:
same response as for `keyexpr_intersect`